pkgs/applications/blockchains/ledger-live-desktop/default.nix

```diff
@@ -2,12 +2,12 @@
 
 let
   pname = "ledger-live-desktop";
-  version = "2.41.3";
+  version = "2.42.0";
   name = "${pname}-${version}";
```

Comment:
This should be converted to pname+version only to fix the output with nix search

Reply:
Done 👍